src/Pages/Login/Components/SignIn.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import './SignIn.scss';
+import { withRouter } from 'react-router-dom';
+
+class SignIn extends Component {
+  state = {
+    emailCheck: '',
+    pwCheck: '',
+  };
+
+  goToAccount = e => {
+    e.preventDefault();
+    const { emailCheck, pwCheck } = this.state;
+    fetch('http://192.168.0.3:8000/users/signin', {
```

Comment:
백엔드 붙이기 시작하셨군요!
백엔드 서버 주소는 자주 바뀌어서 상수로 관리해주시는게 좋습니다!
그리고 상수는 상수라는 걸 나타내기 위해서 대문자_대문자 컨벤션으로 사용합니다!
```suggestion
    fetch(`${SERVER}/users/signin`, {
```

Reply:
넵! 감사합니다!